getting_started/step_by_step/5_adding_items.rst

```diff
@@ -44,86 +44,86 @@ this right now:
 
-Try it out. Depending on which UI you chose, select the look verb or the look
-cursor and click on the Blackboard. Graham should walk there and the text
-should be displayed.
+Try it out! Depending on which UI you chose, select the look verb or the look
+cursor and click on the blackboard. Graham should now walk there and the text
+above should be displayed.
 
 Making the player walk behind items
 -----------------------------------
 
-The other thing we should optimize is making Graham move *behind* the barrels
-instead over it to achieve a three dimensional effect.
+The other thing we should address is making Graham move *behind* the barrels
+instead of overtop of them in order to achieve a three-dimensional effect.
 
-For this, we added an asset that only includes the barrels on a transparent
+For this, we've added an asset that only includes the barrels on a transparent
 background.
 
 Again, add a new node to the ``ESCRoom`` node and select
 ``ESCItem`` as the node type. Name it "Barrels". Add a ``TextureRect`` node to
 this new item and set the texture to the foreground barrels picture.
 
-Move the new items so it is directly over the barrels in the background. You
+Move the new item so it is directly over the barrels in the background. You
 can use the arrow keys of your keyboard for more precision.
 
-To make Graham walk behind the barrels, Escoria uses a little trick.
-The property "z-index" is used by Godot to know which object to draw first.
+To make Graham walk behind the barrels, Escoria uses a little trick: 
+The property ``z-index`` is used by Godot to know which object to draw first.
 
-Objects with a low z-index are drawn first, objects with a high z-index are
-drawn last, so they are drawn over the ones with a low z-index (
-in addition to the node order in the scene tree where the bottom-most nodes
-are drawn last).
+Objects with a lower z-index are drawn first while objects with a higher z-index 
+are drawn last which are drawn over the ones with a lower z-index (
+this is in addition to the node order in the scene tree where the bottom-most 
+nodes are drawn last).
 
-Escoria constantly updates the parameter "z-index" of the player character to
-its y axis. This makes it possible to define objects where the player walks
-behind *and* in front of depending on its y position.
+Escoria constantly updates the parameter ``z-index`` of the player character to
+match the value of its y axis. This makes it possible to define objects where the 
+player walks behind *and* in front of depending on its y position.
 
 See this graphic:
 
 .. image:: img/adding_items_zy.png
    :alt: A visual representation of the following explanation
 
 When the character is at y position 95, it stands behind the table. At y
-position 105 it stands in front of the table. To achieve this effect in
+position 105, it stands in front of the table. To achieve this effect in
 Escoria, we simple set the "z-index" property of the table item to 100.
```

Comment:
Just reading this myself. Shouldn't "simply" be used here?

Reply:
Yep, absolutely. Will fix.